R/await.R

```diff
@@ -124,13 +125,14 @@ await_all <- function(f, .x, ...,
   start <- Sys.time()
   fname <- as.character(substitute(f))
 
-  repeat {
-    responses[!called] <- lapply(.x[!called], safe_call_once,
-                                 f = f, ..., .status_key = .status_key,
-                                 .success_states = .success_states,
-                                 .error_states = .error_states,
-                                 fname = fname)
+  zipped_parameters <- if (is.null(.y)) .x else as.list(paste(.x, .y))
 
+  repeat {
+    responses[!called] <- lapply(zipped_parameters[!called], safe_call_once,
+                                   f = f, ..., .status_key = .status_key,
+                                   .success_states = .success_states,
+                                   .error_states = .error_states,
+                                   fname = fname)
```

Comment:
very small nitpick, but the args `f`, `.success_states`, etc should be aligned with `zipped_parameters`.